test/driver_test.rb

```diff
@@ -1,6 +1,6 @@
 require_relative 'test_helper'
 
-xdescribe "Driver class" do
+describe "Driver class" do
   describe "Driver instantiation" do
     before do
```

Comment:
Nice use of a before block. Consider moving this into the main `decsribe "Driver class" do` so that it can be used by all tests unless there is a reason you need to create a different driver for other tests.